src/langcheck/metrics/en/reference_based_text_quality.py

```diff
@@ -232,7 +233,7 @@ def _rouge(generated_outputs: List[str], reference_outputs: List[str],
     assert rouge_type in ["rouge1", "rouge2", "rougeLsum"]
     scorer = rouge_scorer.RougeScorer([rouge_type], use_stemmer=True)
     scores = []
-    for gen, ref in zip(generated_outputs, reference_outputs):
+    for gen, ref in tqdm_wrapper(zip(generated_outputs, reference_outputs), total=len(generated_outputs)):
```

Comment:
I think for consistency we should show the progress bar in all metrics computations. Could you add mini-batching to the other metrics such as local toxicity, and show the progress bar based on the mini batches?

Reply:
sure, it's a nice idea

Reply:
@liwii I added mini batch, could you take a look